src/verifiers/openAttestationEthereumDocumentStoreIssued.ts

```diff
@@ -0,0 +1,46 @@
+import { getData, v2, v3, WrappedDocument } from "@govtechsg/open-attestation";
+import { isWrappedV3Document, VerificationFragmentType, Verifier } from "../types/core";
+import { verifyIssued } from "../issued/verify";
+import { getDocumentStoreSmartContract } from "../common/smartContract/documentToSmartContracts";
+
+const name = "OpenAttestationEthereumDocumentStoreIssued";
+const type: VerificationFragmentType = "DOCUMENT_STATUS";
+export const openAttestationEthereumDocumentStoreIssued: Verifier<
+  WrappedDocument<v2.OpenAttestationDocument> | WrappedDocument<v3.OpenAttestationDocument>
+> = {
+  skip: () => {
+    return Promise.resolve({
+      status: "SKIPPED",
+      type,
+      name,
+      message: `Document issuers doesn't have "documentStore" or "certificateStore" property or ${v3.Method.DocumentStore} method`
+    });
+  },
+  test: document => {
+    if (isWrappedV3Document(document)) {
+      const documentData = getData(document);
+      return documentData.proof.method === v3.Method.DocumentStore;
+    }
+    const documentData = getData(document);
+    return documentData.issuers.every(issuer => "documentStore" in issuer || "certificateStore" in issuer);
```

Comment:
Should we test using `some` instead? 
This will make the verify run for documents that mixes around documentStore and other methods, and that's where this function should throw?

Reply:
done, returning error if one of the issuer is invalid (i.e. doesnt have document store or certificate store)